### DISCUSSION TITLE
Does grpc-web works over http 1.1?

### DISCUSSION THREAD

**Discussion by wszydlak** [grpc/grpc-web#1390] [Q&A - General]
Does grpc-web works over http 1.1? Or it requires http2?

If it works on http 1.1, then what about bidirectional streaming, and streaming from server when using http 1.1?

**Comment by wszydlak** [comment]
@stanley-cheung @sampajano

**Reply by sampajano** [reply]
Hi :)

It does work over HTTP/1.1 (as documented [here](https://github.com/grpc/grpc/blob/master/doc/PROTOCOL-WEB.md))

Server streaming works since HTTP/1.1 supports it.

Bidirectional streaming is not yet supported. (It's planned for the future using WebTransport.)
